src/main/java/uk/gov/hmcts/darts/audit/api/AuditActivity.java

```diff
@@ -1,11 +1,11 @@
-package uk.gov.hmcts.darts.audit.enums;
+package uk.gov.hmcts.darts.audit.api;
```

Comment:
Why do we want to change the package from `enums` and the enum name?
Only saying because we have it the other way in more places as we like a subpackage

Reply:
I wanted to keep it within the "api" package since they are used by other packages internally. Should I add a sub-package so it would be `audit.api.enums`?

Reply:
I'm happy to keep it as is - I get annoyed at the best of time with our love for subpackages.
Seems fine to me!